packer/resources/features/elk-stack/install.sh

```diff
@@ -18,7 +18,18 @@ sleep 1
 ## Update index and install packages
 apt-get update
 apt-get --yes --force-yes install ruby ruby-dev logstash elasticsearch=1.4.4 \
-    nodejs python-pip golang
+    nodejs python-pip golang libwww-perl libdatetime-perl nginx
+
+## Install Cloudwatch monitoring scripts
+wget "http://aws-cloudwatch.s3.amazonaws.com/downloads/CloudWatchMonitoringScripts-1.2.1.zip" -O /tmp/CloudWatchMonitoringScripts.zip
+unzip /tmp/CloudWatchMonitoringScripts.zip -d /tmp/
+rm /tmp/CloudWatchMonitoringScripts.zip
+mv /tmp/aws-scripts-mon /usr/local/
+
+# Add disk space cron job
+crontab -u root - << EOM
+*/5 * * * * /usr/local/aws-scripts-mon/mon-put-instance-data.pl --disk-space-util --mem-util --disk-path=/ --disk-path=/data --auto-scaling --from-cron
+EOM
```

Comment:
[How media-service does this.](https://github.com/guardian/media-service/blob/master/cloud-formation/media-service-ami.sh#L23)

As its baked into the AMI, we also additionally [use it in the CF template](https://github.com/guardian/grid-infra/blob/master/platform/cloud-formation/media-service.json#L663) to monitor the disk space on the EBS volumes attached to the elastic search instances.